distribution/sender/routes/send-notification-email-route.xml

```diff
@@ -0,0 +1,20 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
```

Comment:
How different are these routes? isn't it possible to create a single route that is used by both the sender and receiver?

Reply:
The only difference is that in the sender we know the senderId from the properties file. So the email cames with the sender name

Reply:
And were should I put this shared route in case we dicide to use the same route? In the sender routes or in the receiver routes? Or should we create a new directory for shared routes?

Reply:
A simple  approach is to move this logic to a processor instead of a route, the processor should be in the **camel-openmrs** sub module, then the processot can be called from both the sender and receiver

Reply:
Okay. I'll do that